packages/graphiql/tsconfig.esm.json

```diff
@@ -11,7 +11,7 @@
       "src/*": ["*"]
     }
   },
-  "include": ["src"],
+  "include": ["src", "src/**/*.json"],
```

Comment:
`.json` files should always be included if `resolveJSONModules` is enabled, right?

Reply:
AFAIK it works even if we don't include it